### PR TITLE
Chore/prisma multifile schema

### PR DESCRIPTION
**Pull request by DecampsRenan**
## Describe your changes

Upgrade prisma to 5.15.1 to be able to split schema file.

Also fix typo and remove version field from docker-compose.yml file (it's not mandatory anymore)

## Documentation

PR to update doc: https://github.com/BearStudio/start-ui-web-docs/pull/52

## Checklist

 - [x] I performed a self review of my code
 - [x] I ensured that everything is written in English
 - [x] I tested the feature or fix on my local environment
 - [x] I ran the `pnpm storybook` command and everything is working
 - [x] If applicable, I created a PR or an issue on the [documentation repository](https://github.com/bearstudio/start-ui-web-docs/): https://github.com/BearStudio/start-ui-web-docs/pull/52

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-version: '3.8'
```
Comment (yoannfleurydev) [inline docker-compose.yml:1]: Is it a good practice to remove the version? I mean, how can `docker compose` know what is the version of the file? Some configuration changes over the versions, so I questioning this change.

Comment (yoannfleurydev) [reply docker-compose.yml:1]: Oh, I so the justification in the commit.

Comment (yoannfleurydev) [reply docker-compose.yml:1]: It's nice they changed that behavior.

Comment (DecampsRenan) [reply docker-compose.yml:1]: I agree, I was a bit worried but it seems it is now the way to go; they keep it to be backward compatible but they recommend to remove it (docker compose will always assume you are on the latest version)
See: https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete

> Compose doesn't use version to select an exact schema to validate the Compose file, but prefers the most recent schema when it's implemented.

I'm not sure it is a good move from them, but this is how it works now 😅

**Review by yoannfleurydev — APPROVED** [review]

**Pull request merged**